src/style.css

```diff
@@ -1 +1,89 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  --background-color: #e5e5e5;
+  --header-color: #fff;
+  --header-height: 60px;
+  --logo-height: 40px;
+  --logo-width: 40px;
+  --link-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  background-color: var(--background-color);
+}
+
+.header {
+  display: flex;
+  position: relative;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--header-color);
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
+}
+
+.logo {
+  padding: 10px 0 10px 50px;
+}
+
+.logo__img {
+  display: block;
+  height: var(--logo-height);
+  width: var(--logo-width);
```

Comment:
as the logo width and height have the same params you can use just one value instead (logo-size)